tests/aws/services/apigateway/test_apigateway_sqs.py

```diff
@@ -250,7 +250,7 @@ def invoke_api(url, is_valid_xml=None):
         patchOperations=[{"op": "replace", "path": "/deploymentId", "value": deployment["id"]}],
     )
 
-    response = retry(invoke_api, sleep=2, retries=10, url=invocation_url, is_valid_xml=is_valid_xml)
+    response = retry(invoke_api, sleep=2, retries=10, url=invocation_url, validate_xml=True)
```

Comment:
:confused: How did that work before, was it just referencing a function, so it would evaluate as `True`? Nice catch!

Reply:
I honestly have no idea how it worked 😅 I think it was overriding and kinda working somehow, but it's nicer like this 😬